### DISCUSSION TITLE
请问如何使用AI扩展节点功能？

### DISCUSSION THREAD

**Discussion by JAMERES86** [graphif/project-graph#224] [问题]
如题，是需要在设置里的哪个地方输入自己的API KEY吗？不过我好像没有找到在哪里有。
使用的是官网上下载的开发版本，系统为Windows。

**Comment by zty012** [comment]
选中一个节点，在右侧工具栏里面点击“AI扩展节点”

**Reply by JAMERES86** [reply]
您好，我这样操作的话，只会拓展出来一个文本内容为error的节点

**Reply by Littlefean** [reply]
欠费了，所以Error了。目前没有开放自定义API KEY

**Comment by Littlefean** [comment]
如果希望大模型赋能思维导图生成，可以先用纯文本导出和纯文本导入两个功能，一个在菜单栏，一个在工具栏。
前两天刚支持了通过网状结构生成，就是需要切换和大模型的对话窗口手动复制一下文字
